### DISCUSSION TITLE
beta 3

### DISCUSSION THREAD

**Discussion by vstony2005** [tamlok/vnote#1634] [General]
Hello,
I just tested version 3.
I can't find a search function on all the notes, as it existed in version 2.
In version 2 you could display folders and notes in 2 different panels, it doesn't seem to be possible at the moment.
The version is in beta, everything may not be finished yet.
In any case, thank you for this extremely practical application, which I use every day.
Good continuation.

**Comment by tamlok** [comment]
Hi,

1. Full-text search is in migration. Check the github Projects for progress.
2. By default, VNote will combine the folders and notes in one panel, to make extra space for the content area. Will keep monitoring the user experience to decide if we need separate panels.

Thanks!

**Reply by vstony2005** [reply]
OK, thank you.
Personally, I was using the 2 panels display which allowed me to access the notes faster.
